src/test/scala/org/allenai/deep_qa/data/OmnibusDaDatasetReaderSpec.scala

```diff
@@ -9,33 +9,33 @@ class OmnibusDaDatasetReaderSpec extends FlatSpecLike with Matchers {
   val questionText1 = "Coal is a nonrenewable energy resource. Identify the original "+
     "source of the energy stored in coal."
   val answerText1 = "The Sun is the original source of energy."
-  val header = "\"id\",\"parentId\",\"isArchived\",\"questionText\"," +
-    "\"answerText\",\"points\",\"isTest\",\"isMultipleChoice\",\"hasDiagram\"," +
-    "\"examName\",\"examGrade\",\"examYear\",\"notes\",\"tags\",\"legacyId\"," +
-    "\"importedQuestionId\""
+  val header = """"id","parentId","isArchived","questionText",""" +
+    """"answerText","points","isTest","isMultipleChoice","hasDiagram",""" +
+    """"examName","examGrade","examYear","notes","tags","legacyId",""" +
+    """importedQuestionId"""
 
-  val row1 = "\"1\",,\"false\",\"" + s"${questionText1}" + "\",\"" + s"${answerText1}" + "\"," +
-    "\"1\",\"false\",\"false\",\"false\",\"Source1\",\"5\",\"2015\",\"\",\"Tag1\"," +
-    "\"legacyId1\",\"importedId1\""
+  val row1 = """"1",,"false","""" + s"${questionText1}" + """","""" + s"${answerText1}" + """",""" +
```

Comment:
oh whoops, the concatenation was a hacky workaround for [a bug in scala that makes escaping not work with string interpolation](https://issues.scala-lang.org/browse/SI-6476). Now that we're using `"""` I agree that it's proper to change it back.